lib/src/creator_utils.dart

```diff
@@ -181,11 +181,19 @@ class DartProjectCreatorTestConfig {
     if (pubspecCreators.isEmpty) {
       name += 'no pubspecs';
     } else {
-      name += pubspecCreators
```

Comment:
This change was unrelated boyscouting to clean up the test output.

Before:
```
00:53 +779: test/....dart: React16_ci_precheck ... with dependencies: [    react: ^4.1.0
]
```
After:
```
00:53 +779: test/....dart: React16_ci_precheck ... with dependencies: [react: any]
```